### DISCUSSION TITLE
non-mp4 video quality for preview

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#6298] [Archive]
Hello! 

Is it possible to watch non-mp4 videos using your file storage?

Best wishes for your team!

**Comment by benhaynes** [comment]
Directus technically supports any file type, and video previews are only limited by browser support.

**Reply by IlyaPetrovM** [reply]
I noticed that files in folder `uploads` stored with different extentions: `.mpga` and `.qt` - what are they?
My mp3 files probably became mpga, and `video.mov` file became `...(uuid).qt` 

Are these formats for making preview?

**Reply by rijkvanzanten** [reply]
> Are these formats for making preview?

No, the filename disk is currently constructed by generating the extension based on the mime type of the file. In this case, `.mov` is actually the new extension for the old QuickTime (`.qt`) format, so the mime type library is defaulting the to "original standard". We'll reverse the logic to rely on the currently existing extension first, and default to the file type if it doesn't exist 👍🏻 

Ref https://github.com/directus/directus/pull/6349

**Reply by IlyaPetrovM** [reply]
Thanks! It's clear now!
